packages/new-polymath-ui/src/components/ModalConfirm/ModalConfirm.tsx

```diff
@@ -32,7 +32,11 @@ const ModalConfirmBase: FC<Props> = props => {
         <Button variant="secondary" onClick={onClose}>
           {cancelButtonText}
         </Button>
-        <ButtonLarge onClick={onSubmit} disabled={isActionDisabled}>
+        <ButtonLarge
+          variant="primary"
```

Comment:
Why specify the variant if it is one of the default props?

Reply:
Yup! I thought I removed all of these